README.md

```diff
@@ -1,4 +1,8 @@
 # Seedvault
+* This branch has been deprecated.
+* Development is now directly done on version branches (such as [android12](https://github.com/seedvault-app/seedvault/tree/android12))
+* Please see https://github.com/seedvault-app/seedvault/issues/301
+
```

Comment:
Maybe use the style we have for `Third-party tools` below which is more visible than the bullets.

Reply:
Done.